paying_for_college/disclosures/scripts/load_programs.py

```diff
@@ -0,0 +1,156 @@
+from __future__ import print_function
+import os
+
+from csvkit import CSVKitDictReader as cdr
```

Comment:
this one's my fault. as much as I **love** csvkit, I don't think we can justify loading it on the servers. I has some heavy dependencies, including SQLAlchemy.

For our use, importing the standard csv library should be fine, although it won't handle encoding oddities as well:

```
from csv import DictReader as cdr
```